run.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, runner):
         self.runner = runner
 
     def format_command(self, wdl_file, json_file, results_file, args, verbose):
-        return self.runner.split(" ") + [wdl_file, "-i", json_file, "-m", results_file] + args
+        return list(filter(None, self.runner.split(" "))) + [wdl_file, "-i", json_file, "-m", results_file] + args
```

Comment:
This might be better written as a comprehension; I'm not sure what `filter(None, ...)` does. Presumably drops falsy elements?